ghost/i18n/locales/pt/portal.json

```diff
@@ -65,109 +65,109 @@
     "Get notified when someone replies to your comment": "Ser notificado quando alguém responde a um comentário teu",
     "Give feedback on this post": "Dar feedback a este artigo",
     "Help! I'm not receiving emails": "Ajuda! Não estou a receber emails",
-    "Here are a few other sites you may enjoy.": "",
-    "If a newsletter is flagged as spam, emails are automatically disabled for that address to make sure you no longer receive any unwanted messages.": "Se uma newsletter for marcada como spam, os emails serão automaticamente desativados para esse endereço para garantir que não receberá mais mensagens indesejadas.",
-    "If the spam complaint was accidental, or you would like to begin receiving emails again, you can resubscribe to emails by clicking the button on the previous screen.": "Se a queixa de spam foi acidental, ou se gostaria de começar a receber emails novamente, pode voltar a subscrever os emails clicando no botão na tela anterior.",
-    "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Se cancelar a sua subscrição agora, continuará a ter acesso até {{periodEnd}}.",
-    "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Se tem uma conta de email corporativa ou governamental, contacte o seu departamento de TI e peça-lhes para permitir a receção de emails de {{senderEmail}}",
+    "Here are a few other sites you may enjoy.": "Aqui estão mais alguns sites que acho que vais gostar.",
+    "If a newsletter is flagged as spam, emails are automatically disabled for that address to make sure you no longer receive any unwanted messages.": "Se uma newsletter for marcada como spam, os emails serão automaticamente desativados para esse endereço para garantir que não receberás mais mensagens indesejadas.",
+    "If the spam complaint was accidental, or you would like to begin receiving emails again, you can resubscribe to emails by clicking the button on the previous screen.": "Se a queixa de spam foi acidental, ou se gostarias de começar a receber emails novamente, podes voltar a subscrever os emails clicando no botão na tela anterior.",
+    "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Se cancelares a tua subscrição agora, continuarás a ter acesso até {{periodEnd}}.",
+    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Se completaste todas estas verificações e ainda não estás a receberes emails, podes contactar {{supportAddress}} para obter apoio.",
```

Comment:
In these files, we only want to update the Portuguese translation. The English part should remain untouched, otherwise the i18n translation mechanism won't work